src/api/objects/ObjectAPI.js

```diff
@@ -623,6 +623,19 @@ export default class ObjectAPI {
             });
     }
 
+    /**
+     * Given an original path check if the path is reachable via root
+     * @param {module:openmct.DomainObject} domainObject the object to check
+     * @returns {boolean} whether the domain object is orphaned
```

Comment:
Think the docs need an update here. If `true`, then the object is reachable, thus not orphaned?